conda_build/ldd.py

```diff
@@ -50,23 +50,21 @@ def get_linkages(obj_files, prefix):
             res[f] = ldd(path)
         elif sys.platform.startswith('darwin'):
             links = otool(path)
-            res[f] = [(basename(l), l) for l in links]
+            res[f] = [(basename(l['name']), l['name']) for l in links]
 
     return res
 
 
 @memoized
 def get_package_obj_files(dist, prefix):
-    with open(join(prefix, 'conda-meta', dist +
-            '.json')) as f:
-        data = json.load(f)
+    data = linked_data(prefix).get(dist)
```

Comment:
Note that we've already read the entire contents of the `conda-meta` directory at this point. Using this command avoids a re-reading.